app/src/main/res/anim/user_approval_status_entry.xml

```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Can't we apply the interpolator in the animation resource?

Reply:
`FastOutSlowInInterpolator()` comes as part of app compat. I don't know how to refer it from XML.